src/App/Routes/Modals/RegisterModal/GithubSignin/index.tsx

```diff
@@ -47,16 +50,21 @@ const GithubSigninContainer: FC<PropsWithChildren<Props>> = ({ children, classNa
         signature,
       });
 
+      toast.success("Github account successfully validated");
+
       setIsRegistering(false);
+      setTxHash(hash);
```

Comment:
The `hash` variable is not a transaction hash but a simple hash of the message (which is a string) we want to sign. It's used by our backend to verify the signature.